.travis.yml

```diff
@@ -16,6 +16,7 @@ addons:
       - python3-pip
       - python3.5
       - g++-7
+      - libssl-dev
```

Comment:
@flub Maybe we should install `libssl1.1-dev`? (to solve #110)

Reply:
Hum, on my debian testing machine there's only `libssl-dev` (installing 1.1.x) and `libssl1.0-dev` which kind of makes sense.  I think with 1.0 we do not get as many compile-time warnings as with 1.1 but it both work afaik.